### ISSUE TITLE
Version without coding ligatures?

### ISSUE THREAD

**Issue by IreneKnapp** [belluzj/fantasque-sans#140]
Hi!

The new coding-centric ligatures in 1.8.0 are really cool and I am certain many people will be drawn to Fantasque Sans because of them.

Could you please add a version of the face without them, for curmudgeons like me who enjoy reading `!=` and `==` and all that quirkiness? As a workaround I can stay on 1.7.2 but it would be nice to know that it'll be supported going forward.

Thank! :)

**Comment by erlliam** [comment]
Agreed, we need a download/version without any ligatures. The ligatures have been ruining the font for me. I will try what you are currently doing (use an older version).

**Comment by erlliam** [comment]
I got a bit interested and I may have found a good solution. I downloaded FontForge and I [followed these instructions](https://stackoverflow.com/questions/43315957/how-to-remove-ligatures-from-a-font-using-fontforge): 

> In FontForge go to Element > Font Info > Lookups > GSUB. You should be able to select a ligature table there. Then you can click Delete to delete all the ligatures or Edit Data to choose which ligatures to delete. 

I deleted everything inside of GSUB lookups and I no longer have ligatures it seems. Not sure if I broke anything though.

![image](https://user-images.githubusercontent.com/32761424/102954815-62ecaa00-44a2-11eb-81b5-ad07e9e1f348.png)
![image](https://user-images.githubusercontent.com/32761424/102954914-ab0bcc80-44a2-11eb-9e00-f092fde82a41.png)

**Comment by IreneKnapp** [comment]
As a note, per the release notes, 1.7.2 is identical to 1.8.0 except that the latter has coding ligatures. So the simpler workaround really is to just use the old version.

**Comment by erlliam** [comment]
I'd say they aren't quite identical, https://github.com/belluzj/fantasque-sans/compare/v1.7.2...v1.8.0. Looks like there might be some problem fixes, can't quite tell from the commit messages. I will just use version 1.7.2 though as I don't like my solution that much.

**Comment by scottmcm** [comment]
(Dup of https://github.com/belluzj/fantasque-sans/issues/122)

**Comment by wcasanova** [comment]
+1, while it can be deactivated.
[Disable ligatures for monospaced fonts](https://wiki.archlinux.org/index.php/Font_configuration/Examples#Disable_ligatures_for_monospaced_fonts)

**Comment by dantezhu** [comment]
+1

**Comment by spinda** [comment]
I've put up a fork at https://github.com/spinda/fantasque-sans-ligatures that strips out everything from v1.8.0 relating to the new coding ligatures.
